analysis/HPC-analysis/dada2_single_script.sh

```diff
@@ -4,7 +4,7 @@
 #SBATCH --cpus-per-task=16
 #SBATCH --time=72:00:00
 
-module load qiime2/2018.6
+module load qiime2/2018.8
```

Comment:
Did you use 2018.6 or 2018.8 for the analysis?

Reply:
8